RealmSwift/Realm.swift

```diff
@@ -563,13 +563,6 @@ public typealias AsyncTransactionId = RLMAsyncTransactionId
         }
     }
 
-    /// :nodoc:
-    @discardableResult
-    @available(*, unavailable, message: "Pass .error, .modified or .all rather than a boolean. .error is equivalent to false and .all is equivalent to true.")
-    public func create<T: Object>(_ type: T.Type, value: Any = [:], update: Bool) -> T {
-        fatalError()
-    }
```

Comment:
This migration helper is nearly four years old and now produces warnings so I just deleted it. It's not a breaking change since you couldn't call it in the first place.